src/inc/internal/AppxBundleWriter.hpp

```diff
@@ -0,0 +1,90 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+// 
+#pragma once
+
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "DirectoryObject.hpp"
+#include "AppxBlockMapWriter.hpp"
+#include "ContentTypeWriter.hpp"
+#include "ZipObjectWriter.hpp"
+#include "BundleWriterHelper.hpp"
+#include "BundleManifestWriter.hpp"
+#include "AppxPackageInfo.hpp"
+
+#include <map>
+#include <memory>
+#include <future>
+
+// internal interface
+// {32e89da5-7cbb-4443-8cf0-b84eedb51d0a}
+#ifndef WIN32
+interface IBundleWriter : public IUnknown
+#else
+#include "Unknwn.h"
+#include "Objidl.h"
+class IBundleWriter : public IUnknown
+#endif
+{
+public:
+    virtual void ProcessBundlePayload(const MSIX::ComPtr<IDirectoryObject>& from, bool flatBundle) = 0;
+
+};
+MSIX_INTERFACE(IBundleWriter, 0x32e89da5,0x7cbb,0x4443,0x8c,0xf0,0xb8,0x4e,0xed,0xb5,0x1d,0x0a);

```

Comment:
>0x32e89da5 [](start = 30, length = 10)

you don't want to copy paste this guid, you should generate a new one

Reply:
Generated and added new guid in here.